client/app/index.js

```diff
@@ -26,6 +34,8 @@ function loadCurrentModule() {
 }
 
 function loadModules() {
+  loadAceEditorDependencies();
```

Comment:
oh oops the default export was because I first put it in another file. forgot to remove it. Anyway I'll put the requires in `layout_ace_editor.js`